src/features/kakao-map/components/marker/MapWithMarkers.tsx

```
@@ -136,6 +253,11 @@
         setInternalSelectedStoreId(store.storeId);
         setInfoWindowStore(store);
 
+        // 추천 매장인지 확인
+        const isRecommended = recommendedStores.some(
```

ESLint

[error] 'isRecommended' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)